### PR TITLE
Multiply numbers issue #165

### PR DESCRIPTION
**Pull request by tanvi15**

### CODE REVIEW

File: Data Structures/Linked-Lists/multiply.cpp [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/* In this program, two numbers, denoted by two linked lists whose
+ * nodes contain the digits, are multiplied. This is done by first
+ * adding the digits to the tail of the linked list and then
+ * extracting each digit from the head to make the complete number.
+ *
+ * The time complexity of this program is O(m*n) where m and n are
```
Comment (vikasgola) [inline Data Structures/Linked-Lists/multiply.cpp]: This should be at the end.

**Comment by tanvi15** [comment]
@vikasgola done

**Comment by tanvi15** [comment]
@ramitsawhney27 done. Please review.

**Comment by tanvi15** [comment]
@ramitsawhney27 please review

**Pull request merged**